.circleci/config.yml

```diff
@@ -35,8 +35,19 @@ jobs:
       - store_artifacts:
           path: cover/
 
+  test26:
+    docker:
+      - image: circleci/python:2.7-jessie-browsers
+    steps:
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - run:
+          command: docker run -it -e TEST_BRANCH=AGENT-93-circle-python-versions edwardchee/scalyr-agent-ci:python26 /tmp/run_unittests.sh
```

Comment:
I'm currently using my own public repo to host this custom docker image. We can later move it under the public Scalyr account but I don't yet have authorization.  No security risk for now (this image has nothing sensitive, just 26-compiled python & pip)

Reply:
Do you not have access to dockerhub from doing the build process?  If you can publish the official scalyr-agent builds, you should be able to add other public repos too.

Also, with regards to this, I think we should change the name of the image slightly.  Common practice generally seems to be to include the tech stack in the name, and use the tag for the version only e.g. assuming the image gets moved to the public scalyr repo, something like `scalyr/scalyr-agent-ci-python:2.6`

Reply:
P.S commands for doing this would be:

`docker login`
(username `scalyr`, password `<you have this already somewhere>`

`docker push scalyr/scalyr-agent-ci-python:2.6`

Reply:
And other thought, do you have the Dockerfile you used to generate this image?  That should probably be added to the main repo too, maybe under the `docker` directory.

Reply:
Let's hold off docker image naming/Dockerfile as those are in flux and might change in future so we should avoid unnecessary code review.  Also not just a single dockerfile but current implementation (which in future might change) requires a .tgz with compiled python binaries -- since this repo is public and really should be for agent-specific code, others cloning this repo shouldn't have to download largish unrelated files.

I'm also starting a separate "ingest devops" private scalyr repo for CD/release automation related files and the above might be better suited there. 

Can discuss more in next standup.

Reply:
BTW goal of this PR is so we can start seeing 26 agent code errors (and then 25, 24) as soon as possible.  For example, once this PR is merged, anyone can start making code fix PRs and immediately how it affects 26. 

So, note that I'm making a conscious decision to defer any devops / docker image issues and optimizations for the future. After 25,24, the top priority is install release automation.

Once we have all automation functionally working, we can chip away at cleanup or things like docker image caching.  For example, we might spend time to find a CircleCI python26 image and replace what I've installed with that.

Reply:
Yep.  Agree this doesn't need to be addressed yet, just wanted to be sure we are thinking about it.